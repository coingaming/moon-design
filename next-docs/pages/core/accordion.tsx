import Preview from '../../components/codePreview/Preview';
import ComponentPageDescription from '../../components/ComponentPageDescription';
import PropsTable from '../../components/PropsTable';
import Background from '../../public/examples/accordionTW/Background';
import ContentHeader from '../../public/examples/accordionTW/ContentHeader';
import ContentOutside from '../../public/examples/accordionTW/ContentOutside';
import ContentOutsideSizes from '../../public/examples/accordionTW/ContentOutsideSizes';
import Default from '../../public/examples/accordionTW/Default';
import Disabled from '../../public/examples/accordionTW/Disabled';
import OpenDefault from '../../public/examples/accordionTW/OpenDefault';
import Sizes from '../../public/examples/accordionTW/Sizes';
import WithoutButton from '../../public/examples/accordionTW/WithoutButton';
import useExamples from '../../utils/useExamples';

const Example = () => {
  const examples = useExamples('accordionTW');
  return (
    <>
      <ComponentPageDescription
        title="Accordion"
        isInProgress
        isRtlSupport
        isAriaSupport
      >
        <p>
          Like the accordion instrument, our accordion component reveals or
          hides associated sections of content. This is done through the use of
          a vertically stacked list of headers.
        </p>
        <p>
          Users can decide which sections to toggle, read and close as the
          header titles will give them a high-level overview of the content
          that's in the space.
        </p>
      </ComponentPageDescription>
      <Preview
        title="Default"
        preview={<Default />}
        code={examples ? examples.Default : 'Loading'}
      />
      <Preview
        title="Open by default"
        preview={<OpenDefault />}
        code={examples ? examples.OpenDefault : 'Loading'}
      />
      <Preview
        title="Without button"
        preview={<WithoutButton />}
        code={examples ? examples.WithoutButton : 'Loading'}
      />
      <Preview
        title="Disabled"
        preview={<Disabled />}
        code={examples ? examples.Disabled : 'Loading'}
      />
      <Preview
        title="Content outside"
        preview={<ContentOutside />}
        code={examples ? examples.ContentOutside : 'Loading'}
      />
      <Preview
        title="Header content"
        preview={<ContentHeader />}
        code={examples ? examples.ContentHeader : 'Loading'}
      />
      <Preview
        title="Sizes"
        preview={<Sizes />}
        code={examples ? examples.Sizes : 'Loading'}
      />
      <Preview
        title="Sizes: content outside"
        preview={<ContentOutsideSizes />}
        code={examples ? examples.ContentOutsideSizes : 'Loading'}
      />
      <Preview
        title="Custom background"
        preview={<Background />}
        code={examples ? examples.Background : 'Loading'}
      />
      <PropsTable
        title="Accordion props"
        data={[
          {
            name: 'bgColor',
            type: 'string',
            required: false,
            default: 'bg-gohan',
            description: 'Alters background colour of accordion',
          },
          {
            name: 'children',
            type: 'React.ReactNode',
            required: false,
            default: '-',
            description: 'Children content',
          },
          {
            name: 'disabled',
            type: 'boolean',
            required: false,
            default: 'false',
            description: 'Disables accordion',
          },
          {
            name: 'headerContent',
            type: 'React.ReactNode',
            required: false,
            default: '-',
            description: 'Additional content to render in accordion header',
          },
          {
            name: 'isContentInside',
            type: 'boolean',
            required: false,
            default: 'true',
            description:
              'Displays content inside of the accordion header block',
          },
          {
            name: 'openByDefault',
            type: 'boolean',
            required: false,
            default: 'false',
            description: 'Expanded accordion by default',
          },
          {
            name: 'size',
            type: 'sm | md | lg | xl',
            required: false,
            default: 'md',
            description: 'Size of accordion',
          },
          {
            name: 'title',
            type: 'string | React.ReactNode',
            required: true,
            default: '-',
            description: 'Title of accordion',
          },
          {
            name: 'withButton',
            type: 'boolean',
            required: false,
            default: 'true',
            description: 'Toggles expand/collapse arrow',
          },
        ]}
      />
    </>
  );
};

export default Example;
