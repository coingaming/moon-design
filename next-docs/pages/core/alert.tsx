import Preview from '../../components/codePreview/Preview';
import ComponentAnatomy from '../../components/ComponentAnatomy';
import ComponentPageDescription from '../../components/ComponentPageDescription';
import getComponent from '../../components/getComponent';
import PropsTable from '../../components/PropsTable';
import Customization from '../../public/examples/alert/Customization';
import Default from '../../public/examples/alert/Default';
import HandleClose from '../../public/examples/alert/HandleClose';
import WithClose from '../../public/examples/alert/WithClose';
import WithIcon from '../../public/examples/alert/WithIcon';
import WithIconAndClose from '../../public/examples/alert/WithIconAndClose';
import WithTitle from '../../public/examples/alert/WithTitle';
import useExamples from '../../utils/useExamples';

const Example = () => {
  const examples = useExamples('alert');
  const { name, text, image } = getComponent('Alert');
  return (
    <>
      <ComponentPageDescription
        title={name}
        image={image}
        isInProgress
        isRtlSupport
        isAriaSupport
      >
        <p>{text}</p>
      </ComponentPageDescription>
      <ComponentAnatomy>
        {`<Alert>
  <Alert.Title>...</Alert.Title>
  <Alert.Message>...</Alert.Message>
  <Alert.Close />
</Alert>`}
      </ComponentAnatomy>
      <Preview
        title="Default"
        preview={<Default />}
        code={examples ? examples.Default : 'Loading'}
      />
      <Preview
        title="With title"
        preview={<WithTitle />}
        code={examples ? examples.WithTitle : 'Loading'}
      />
      <Preview
        title="With icon"
        preview={<WithIcon />}
        code={examples ? examples.WithIcon : 'Loading'}
      />
      <Preview
        title="With close"
        preview={<WithClose />}
        code={examples ? examples.WithClose : 'Loading'}
      />
      <Preview
        title="With icon and close"
        preview={<WithIconAndClose />}
        code={examples ? examples.WithIconAndClose : 'Loading'}
      />
      <Preview
        title="Handle close"
        preview={<HandleClose />}
        code={examples ? examples.HandleClose : 'Loading'}
      />
      <Preview
        title="Customization"
        preview={<Customization />}
        code={examples ? examples.Customization : 'Loading'}
      />
      <PropsTable
        title="Alert props"
        data={[
          {
            name: 'className',
            type: 'string',
            required: false,
            default: '-',
            description: 'Tailwind classes for customization',
          },
        ]}
      />
      <PropsTable
        title="Alert.Title props"
        data={[
          {
            name: 'children',
            type: 'string | React.ReactNode',
            required: false,
            default: '-',
            description: 'Children content',
          },
          {
            name: 'className',
            type: 'string',
            required: false,
            default: '-',
            description: 'Tailwind classes for customization',
          },
        ]}
      />
      <PropsTable
        title="Alert.Message props"
        data={[
          {
            name: 'children',
            type: 'string | React.ReactNode',
            required: false,
            default: '-',
            description: 'Children content',
          },
          {
            name: 'className',
            type: 'string',
            required: false,
            default: '-',
            description: 'Tailwind classes for customization',
          },
        ]}
      />
    </>
  );
};

export default Example;
