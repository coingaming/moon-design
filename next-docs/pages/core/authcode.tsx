import React from 'react';
import Preview from '../../components/codePreview/Preview';
import ComponentPageDescription from '../../components/ComponentPageDescription';
import PropsTable from '../../components/PropsTable';
import Default from '../../public/examples/authCodeTW/Default';
import ErrorState from '../../public/examples/authCodeTW/ErrorState';
import FourChars from '../../public/examples/authCodeTW/FourChars';
import OnlyDigits from '../../public/examples/authCodeTW/OnlyDigits';
import Stretch from '../../public/examples/authCodeTW/Stretch';
import useExamples from '../../utils/useExamples';

const PageAuthCodeGroup = () => {
  const examples = useExamples('authCodeTW');

  return (
    <>
      <ComponentPageDescription title="AuthCode">
        <p>
          The AuthCode component consists of TextInput components with
          additional logic that supports quick and easy one-time-password
          handling.
        </p>
      </ComponentPageDescription>
      <Preview
        title="Default"
        preview={<Default />}
        code={examples ? examples.Default : 'Loading'}
      />
      <Preview
        title="Custom length"
        preview={<FourChars />}
        code={examples ? examples.FourChars : 'Loading'}
      />
      <Preview
        title="Only digits"
        preview={<OnlyDigits />}
        code={examples ? examples.OnlyDigits : 'Loading'}
      />
      <Preview
        title="Error state"
        preview={<ErrorState />}
        code={examples ? examples.ErrorState : 'Loading'}
      />
      <Preview
        title="Stretch"
        preview={<Stretch />}
        code={examples ? examples.Stretch : 'Loading'}
      />
      <PropsTable
        title="Props"
        data={[
          {
            name: 'length',
            type: 'number',
            required: false,
            default: '6',
            description: 'Number of inputs for inserting single values.',
          },
          {
            name: 'onlyDigits',
            type: 'boolean',
            required: false,
            default: 'false',
            description:
              'If true, inputs would allow only digit values. Default is alphanumeric.',
          },
          {
            name: 'stretch',
            type: 'boolean',
            required: false,
            default: 'false',
            description:
              'If true, inputs would get streched to fill the available width',
          },
          {
            name: 'placeholder',
            type: 'string',
            required: false,
            default: '-',
            description: 'Default placeholder for input elements.',
          },
          {
            name: 'errorMessage',
            type: 'string',
            required: false,
            default: '-',
            description:
              'Puts element in error state and displays the message.',
          },
          {
            name: 'onChange',
            type: 'function',
            required: false,
            default: '-',
            description:
              'Callback function that gets triggered on single value change.',
          },
          {
            name: 'onSubmit',
            type: 'function',
            required: false,
            default: '-',
            description:
              'Callback function that gets triggered when all single values are entered.',
          },
        ]}
      />
    </>
  );
};

export default PageAuthCodeGroup;
