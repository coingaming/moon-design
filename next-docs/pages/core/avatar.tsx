import Preview from '../../components/codePreview/Preview';
import ComponentPageDescription from '../../components/ComponentPageDescription';
import Active from '../../public/examples/avatarTW/Active';
import Colours from '../../public/examples/avatarTW/Colours';
import Default from '../../public/examples/avatarTW/Default';
import Rounded from '../../public/examples/avatarTW/Rounded';
import Sizes from '../../public/examples/avatarTW/Sizes';
import StatusOrigin from '../../public/examples/avatarTW/StatusOrigin';
import Variants from '../../public/examples/avatarTW/Variants';
import useExamples from '../../utils/useExamples';

const Example = () => {
  const examples = useExamples('avatarTW');
  return (
    <>
      <ComponentPageDescription title="Avatar">
        <p>
          Avatars are used to represent an individual, a business, or a
          celestial being when they interact with any Coingaming brand. While
          they usually consist of an image, avatars are adaptable - letters or
          object icons may sometimes be used as fallbacks.
        </p>
      </ComponentPageDescription>
      <Preview
        title="Default"
        preview={<Default />}
        code={examples ? examples.Default : 'Loading'}
      />
      <Preview
        title="Variants"
        preview={<Variants />}
        code={examples ? examples.Variants : 'Loading'}
      />
      <Preview
        title="Rounded"
        preview={<Rounded />}
        code={examples ? examples.Rounded : 'Loading'}
      />
      <Preview
        title="Different colours"
        preview={<Colours />}
        code={examples ? examples.Colours : 'Loading'}
      />
      <Preview
        title="Sizes"
        preview={<Sizes />}
        code={examples ? examples.Sizes : 'Loading'}
      />
      <Preview
        title="Active status"
        preview={<Active />}
        code={examples ? examples.Active : 'Loading'}
      />
      <Preview
        title="Status origin"
        preview={<StatusOrigin />}
        code={examples ? examples.StatusOrigin : 'Loading'}
      />
    </>
  );
};

export default Example;
