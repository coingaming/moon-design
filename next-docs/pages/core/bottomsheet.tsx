import React from 'react';
import Preview from '../../components/codePreview/Preview';
import ComponentPageDescription from '../../components/ComponentPageDescription';
import PropsTable from '../../components/PropsTable';
import CustomHeight from '../../public/examples/bottomsheetTW/CustomHeight';
import Default from '../../public/examples/bottomsheetTW/Default';
import LargeWithOptionalExtras from '../../public/examples/bottomsheetTW/LargeWithOptionalExtras';
import useExamples from '../../utils/useExamples';

const Example = () => {
  const examples = useExamples('bottomsheetTW');
  return (
    <>
      <ComponentPageDescription title="Bottomsheet" isInProgress>
        <p>
          A Bottomsheet is an interface element that appears over other content.
          It requires an interaction from the user before they can return to
          whatever is underneath. It is different from a modal in that it's
          always anchored to the bottom of the viewport and can be slid away
          downwards to collapse it. It is designed for small mobile screens.
        </p>
      </ComponentPageDescription>
      <Preview
        title="Large with all optional extras"
        preview={<LargeWithOptionalExtras />}
        code={examples ? examples.LargeWithOptionalExtras : 'Loading'}
      />
      <Preview
        title="Default"
        preview={<Default />}
        code={examples ? examples.Default : 'Loading'}
      />
      <Preview
        title="Custom height"
        preview={<CustomHeight />}
        code={examples ? examples.CustomHeight : 'Loading'}
      />
      <PropsTable
        title="Bottomsheet"
        data={[
          {
            name: 'open',
            type: 'boolean',
            required: true,
            default: '-',
            description: 'Whether the Bottomsheet is open or not.',
          },
          {
            name: 'onClose',
            type: '() => void',
            required: true,
            default: '-',
            description: 'Called when the Bottomsheet is dismissed.',
          },
          {
            name: 'hasShadow',
            type: 'boolean',
            required: false,
            default: '-',
            description: 'Whether the Bottomsheet has a shadow or not.',
          },
          {
            name: 'size',
            type: 'string',
            required: false,
            default: 'sm',
            description:
              'The size of the Bottomsheet, sm, md, lg or custom pixel/percentage/rem value.',
          },
        ]}
      />
      <PropsTable
        title="Bottomsheet.Title"
        data={[
          {
            name: 'className',
            type: 'string',
            required: false,
            default: '-',
            description:
              'Tailwind classes for customization of Bottomsheet Title.',
          },
        ]}
      />
      <PropsTable
        title="Bottomsheet.Panel"
        data={[
          {
            name: 'className',
            type: 'string',
            required: false,
            default: '-',
            description:
              'Tailwind classes for customization of Bottomsheet Panel.',
          },
        ]}
      />
      <PropsTable
        title="Bottomsheet.Backdrop"
        data={[
          {
            name: 'className',
            type: 'string',
            required: false,
            default: '-',
            description:
              'Tailwind classes for customization of Bottomsheet Backdrop.',
          },
        ]}
      />
    </>
  );
};

export default Example;
