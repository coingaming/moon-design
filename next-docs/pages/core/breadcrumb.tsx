import Preview from '../../components/codePreview/Preview';
import ComponentPageDescription from '../../components/ComponentPageDescription';
import PropsTable from '../../components/PropsTable';
import Collapsed from '../../public/examples/breadcrumbs/Collapsed';
import FourItems from '../../public/examples/breadcrumbs/FourItems';
import OneItem from '../../public/examples/breadcrumbs/OneItem';
import TwoItems from '../../public/examples/breadcrumbs/TwoItems';
import useExamples from '../../utils/useExamples';

const Example = () => {
  const examples = useExamples('breadcrumbs');
  return (
    <>
      <ComponentPageDescription title="Breadcrumb" isInProgress isRtlSupport>
        <p>
          A list of links showing the location of the current page in the
          navigational hierarchy.
        </p>
      </ComponentPageDescription>
      <Preview
        title="Five and more items"
        preview={<Collapsed />}
        code={examples ? examples.Collapsed : 'Loading'}
      />
      <Preview
        title="Four items"
        preview={<FourItems />}
        code={examples ? examples.FourItems : 'Loading'}
      />
      <Preview
        title="Two items"
        preview={<TwoItems />}
        code={examples ? examples.TwoItems : 'Loading'}
      />
      <Preview
        title="One item"
        preview={<OneItem />}
        code={examples ? examples.OneItem : 'Loading'}
      />
      <PropsTable
        title="Breadcrumb props"
        data={[
          {
            name: 'breadcrumbs',
            type: 'React.ElementType[]',
            required: true,
            default: '-',
            description: 'Contents of breadcrumb',
          },
        ]}
      />
    </>
  );
};

export default Example;
