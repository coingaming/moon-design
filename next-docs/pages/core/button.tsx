import React from 'react';
import Preview from '../../components/codePreview/Preview';
import Animations from '../../public/examples/buttonTW/Animations';
import Disabled from '../../public/examples/buttonTW/Disabled';
import FullWidth from '../../public/examples/buttonTW/FullWidth';
import Icons from '../../public/examples/buttonTW/Icons';
import Sizes from '../../public/examples/buttonTW/Sizes';
import Variants from '../../public/examples/buttonTW/Variants';
import useExamples from '../../utils/useExamples';

const PageButton = () => {
  const examples = useExamples('buttonTW');

  return (
    <div className="theme-moon-dark">
      <section className="mt-8">
        <Preview
          title="Variants"
          preview={<Variants />}
          code={examples ? examples.Variants : 'Loading'}
        />
      </section>

      <section className="mt-8">
        <Preview
          title="Sizes"
          preview={<Sizes />}
          code={examples ? examples.Sizes : 'Loading'}
        />
      </section>

      <section className="mt-8">
        <Preview
          title="Icons"
          preview={<Icons />}
          code={examples ? examples.Icons : 'Loading'}
        />
      </section>

      <section className="mt-8">
        <Preview
          title="Full width"
          preview={<FullWidth />}
          code={examples ? examples.FullWidth : 'Loading'}
        />
      </section>

      <section className="mt-8">
        <Preview
          title="Disabled"
          preview={<Disabled />}
          code={examples ? examples.Disabled : 'Loading'}
        />
      </section>

      <section className="mt-8">
        <Preview
          title="Animations"
          preview={<Animations />}
          code={examples ? examples.Animations : 'Loading'}
        />
      </section>
    </div>
  );
};

export default PageButton;
