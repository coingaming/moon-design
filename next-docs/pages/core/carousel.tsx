import React from 'react';
import Preview from '../../components/codePreview/Preview';
import ComponentAnatomy from '../../components/ComponentAnatomy';
import ComponentPageDescription from '../../components/ComponentPageDescription';
import PropsTable from '../../components/PropsTable';
import CustomizedArrow from '../../public/examples/carousel/CustomizedArrow';
import Default from '../../public/examples/carousel/Default';
import Indicators from '../../public/examples/carousel/Indicators';
import SelectIndex from '../../public/examples/carousel/SelectIndex';
import Spaces from '../../public/examples/carousel/Spaces';
import VisibleIndex from '../../public/examples/carousel/VisibleIndex';
import useExamples from '../../utils/useExamples';

export default function PageCarousel() {
  const examples = useExamples('carousel');

  return (
    <>
      <ComponentPageDescription title="Carousel" isInProgress>
        <p>
          Is an effective way of displaying multiple images or content in a
          single space.
        </p>
        <p>
          It not only helps in saving screen space, but also encourages visitors
          to focus on important website content and improves the overall visual
          appeal effectively.
        </p>
      </ComponentPageDescription>
      <ComponentAnatomy>
        {`<Carousel>
  <Carousel.LeftArrow>...</Carousel.LeftArrow>
  <Carousel.Reel>
    <Carousel.Item>...</Carousel.Item>
  <Carousel.Reel>
  <Carousel.Indicators/>
  </Carousel.RightArrow>...</Carousel.RightArrow>
</Carousel>`}
      </ComponentAnatomy>
      <Preview
        title="Carousel"
        preview={<Default />}
        code={examples ? examples.Default : 'Loading'}
      />
      <Preview
        title="Customized Arrows"
        preview={<CustomizedArrow />}
        code={examples ? examples.CustomizedArrow : 'Loading'}
      />
      <Preview
        title="Indicators"
        preview={<Indicators />}
        code={examples ? examples.Indicators : 'Loading'}
      />
      <Preview
        title="Spaces"
        preview={<Spaces />}
        code={examples ? examples.Spaces : 'Loading'}
      />
      <Preview
        title="Visible indices"
        preview={<VisibleIndex />}
        code={examples ? examples.VisibleIndex : 'Loading'}
      />
      <Preview
        title="Select index"
        preview={<SelectIndex />}
        code={examples ? examples.SelectIndex : 'Loading'}
      />
      <PropsTable
        title="Carousel props"
        data={[
          {
            name: 'scrollTo',
            type: 'number',
            required: false,
            default: '-',
            description: 'Index of item to scroll to',
          },
          {
            name: 'step',
            type: 'number',
            required: false,
            default: '5',
            description: 'Step of scroll',
          },
          {
            name: 'selectedIndex',
            type: 'number',
            required: false,
            default: '-',
            description: 'Index of selected item',
          },
          {
            name: 'Render Props: ',
            type: '',
            required: undefined,
            default: '',
            description: '',
          },
          {
            name: 'scrollLeftToStep',
            type: '() => void',
            required: false,
            default: '-',
            description: 'Scrolls left',
          },
          {
            name: 'scrollRightToStep',
            type: '() => void',
            required: false,
            default: '-',
            description: 'Scrolls right',
          },
          {
            name: 'canScrollLeft',
            type: 'boolean',
            required: false,
            default: '-',
            description: 'Whether can scroll left',
          },
          {
            name: 'canScrollRight',
            type: 'boolean',
            required: false,
            default: '-',
            description: 'Whether can scroll right',
          },
          {
            name: 'firstVisibleIndex',
            type: 'number',
            required: false,
            default: '-',
            description: 'Index of the first visible item',
          },
          {
            name: 'lastVisibleIndex',
            type: 'number',
            required: false,
            default: '-',
            description: 'Index of the last visible item',
          },
        ]}
      />
    </>
  );
}
