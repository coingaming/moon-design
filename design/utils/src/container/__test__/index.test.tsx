/** @jsx jsx */
import { jsx } from '@emotion/core';
import { create } from 'react-test-renderer';
import { container } from '../';

describe('Container', () => {
  test('renders correctly by default', () => {
    expect(create(<div css={container()} />)).toMatchSnapshot();
  });

  test('renders container key', () => {
    expect(create(<div css={container('large')} />)).toMatchSnapshot();
  });

  test('renders a rem-based container with number value', () => {
    expect(create(<div css={container(300)} />)).toMatchSnapshot();
  });
});
