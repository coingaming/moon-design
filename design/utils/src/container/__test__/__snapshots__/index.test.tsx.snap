// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders a rem-based container with number value 1`] = `
.emotion-0 {
  padding-right: 1rem;
  padding-left: 1rem;
  margin-right: auto;
  margin-left: auto;
  max-width: 18.75rem;
}

<div
  className="emotion-0"
/>
`;

exports[`Container renders container key 1`] = `
.emotion-0 {
  padding-right: 1rem;
  padding-left: 1rem;
  margin-right: auto;
  margin-left: auto;
  max-width: 90rem;
}

<div
  className="emotion-0"
/>
`;

exports[`Container renders correctly by default 1`] = `
.emotion-0 {
  padding-right: 1rem;
  padding-left: 1rem;
  margin-right: auto;
  margin-left: auto;
  max-width: 64rem;
}

<div
  className="emotion-0"
/>
`;

exports[`Container renders with custom padding key 1`] = `
.emotion-0 {
  padding-right: 0.5rem;
  padding-left: 0.5rem;
  margin-right: auto;
  margin-left: auto;
  max-width: 64rem;
}

<div
  className="emotion-0"
/>
`;

exports[`Container renders with custom padding number value 1`] = `
.emotion-0 {
  padding-right: 3.125rem;
  padding-left: 3.125rem;
  margin-right: auto;
  margin-left: auto;
  max-width: 64rem;
}

<div
  className="emotion-0"
/>
`;

exports[`Container renders with no padding 1`] = `
.emotion-0 {
  padding-right: 0rem;
  padding-left: 0rem;
  margin-right: auto;
  margin-left: auto;
  max-width: 64rem;
}

<div
  className="emotion-0"
/>
`;
